scripts/qemu/auto_qemu_with_usb_storage

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+image_file_name="usbdisk.img"
+
+if [ -f "$file" ]
+then
+	echo "$image_file_name found."
+else
+	echo "$image_file_name not found."
+	qemu-img create -f qcow2 usbdisk.img 512M
+fi
+
+sudo mkfs -t vfat $image_file_name
```

Comment:
It's not good to have `sudo` in shell scripts. It's better to use sudo when calling script.